scripts/download/wsl.mjs

```diff
@@ -14,4 +15,51 @@ export default async function main() {
     `https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/releases/download/v${ v }/distro-${ v }.tar`,
     path.resolve(process.cwd(), 'resources', os.platform(), `distro-${ v }.tar`),
     { access: fs.constants.W_OK });
+
+  // Download host-resolver
+  // TODO(@Nino-k) once host-resolver stabilizes remove and add to wsl-distro
+  downloadHostResolver();
+}
+
+async function extract(resourcesPath, file, expectedFile) {
+  const bsdTar = path.join(process.env.SystemRoot, 'system32', 'tar.exe');
+  const arg = [bsdTar, '-xzf', file, expectedFile];
+
+  await spawnSync(
+    arg[0],
+    arg.slice(1),
```

Comment:
Optional:
```suggestion
  await spawnSync(
    bsdTar,
    ['-xzf', file, expectedFile],
```
I just have no idea why we need to do the extra array; it's fine if you want to keep it.